src/components/AlphanumericInput/AlphanumericInput.svelte

```diff
@@ -60,11 +61,11 @@
   }
 </script>
 
-<div class={`numeric-input d-flex ${className}`}>
+<div class={`d-flex ${className}`}>
```

Comment:
A component should always have a class with his name:

```
<div class={`alphanumeric-input d-flex ${className}`}>
```

What was the reason of the deletion? Please first ask next time so we can discuss it 😅

Reply:
>A component should always have a class with his name

If any class is needed, we don't need it :thinking:

Reply:
Yes we do to easily access the component globally as a convention.

Reply:
>Yes we do to easily access the component globally as a convention. So that the class is like a namespace for each component

But hat happens if the component is instantiated multiple times ? There is a conflict between the different classes.

Thus it is a bad practice to do it with a class: we should not access the component based on a class!


Again here, a class is not needed.